examples/examples/z_ping.rs

```diff
@@ -90,12 +94,13 @@ struct Args {
     common: CommonArgs,
 }
 
-fn parse_args() -> (Config, Duration, usize, usize) {
+fn parse_args() -> (Config, Duration, usize, usize, bool) {
     let args = Args::parse();
     (
         args.common.into(),
         Duration::from_secs_f64(args.warmup),
         args.payload_size,
         args.samples,
+        !args.no_express,
```

Comment:
Because in clap a boolean argument is always true if set. I.e.: `--express` would be `true` if passed and it is not possible to have `--express false`.